src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+
+export default function Dropdown(props) {
+	const [isExpanded, setIsExpanded] = useState(true);
+  const {label, dropdown, submenu} = props;
+	function dropMenu () {
+		setIsExpanded(!isExpanded);
+	}
+	const dropClass = isExpanded ? "dropdown__content-off" : "dropdown__content";
```

Comment:
This is wrong, the name of this element should be constantly added but modified with a BEM fix or BEM modifier - like:

But overall - you can try not to render this part depending on `isExpanded` like:
```
{isExpanded && (
 <div className="dropdown__content">
...
 </div>
)}
```